paseos/utils/operations_monitor.py

```diff
@@ -0,0 +1,81 @@
+import csv
+
+from loguru import logger
+from dotmap import DotMap
+import pykep as pk
+
+from paseos.actors.base_actor import BaseActor
+from paseos.actors.spacecraft_actor import SpacecraftActor
+
+
+class OperationsMonitor:
+    """This class is used to track actor status and activities over time."""
+
+    def __init__(self, actor_name):
+        """Initializes the OperationsMonitor
+
+        Args:
+            actor_name (str): Name of the local actor.
+        """
+        logger.trace("Initializing OperationsMonitor for " + actor_name)
+        self._actor_name = actor_name
+        self._log = DotMap(_dynamic=False)
+        self._log.timesteps = []
+        self._log.current_activity = []
+        self._log.battery_ratio = []
```

Comment:
What do you mean with ratio? Current Energy / Total Energy? Can you call it "State of charge" https://en.wikipedia.org/wiki/State_of_charge